texar/torch/modules/encoders/bert_encoders.py

```diff
@@ -41,15 +41,16 @@ class BERTEncoder(EncoderBase, PretrainedBERTMixin):
     :class:`~texar.torch.modules.encoders.TransformerEncoder` and a dense
     pooler.
 
-    This module supports the architecture first proposed
-    in `(Devlin et al.)` BERT.
+    This module supports the architectures first proposed
+    in `(Devlin et al.)` BERT and `(Liu et al.)` RoBERTa.
 
     Args:
         pretrained_model_name (optional): a str with the name
             of a pre-trained model to load selected in the list of:
             `bert-base-uncased`, `bert-large-uncased`, `bert-base-cased`,
             `bert-large-cased`, `bert-base-multilingual-uncased`,
-            `bert-base-multilingual-cased`, `bert-base-chinese`.
+            `bert-base-multilingual-cased`, `bert-base-chinese`,
+            `roberta-base`, `roberta-large`, `roberta-large-mnli`.
```

Comment:
Make the list more structured, and refer users to the above structured note for more details.